.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,8 @@
 <!-- Thank you for opening a PR! We really appreciate you taking the time to help out 🙌 -->
 
 #### Description (required)
-<!-- Please describe the change you are proposing, and why -->
```

Comment:
I don't think this portion should be deleted 🤔

Reply:
That part wasn't deleted, it is still there 😁

Reply:
Ohh I saw the Deletion (Red background thing) and completely missed the Green background 🤦